examples/pytorch/llm/scripts/qwen_72b_chat/torchacc/full_fsdp_sft.sh

```diff
@@ -0,0 +1,34 @@
+# Experimental environment: 4 * 8*A100
+# 80GB GPU memory
+# Note: TorchAcc is currently only available internally.
```

Comment:
@baoleai when can we use TorchAcc?

Reply:
Thank you for your interest. We are currently undergoing comprehensive testing and optimization in multiple scenarios. Once testing is complete, we will release a customized runtime environment for TorchAcc with detailed deployment and usage guidelines. Ultimately, we will open source TorchAcc with rigorous testing and clean code.